src/microbe_stage/ProcessSystem.cs

```diff
@@ -43,6 +43,15 @@ public ProcessSystem(Node worldRoot)
         return result;
     }
 
+    /// <summary>
+    ///   Computes the energy balance for the given organelles in biome
+    /// </summary>
+    public static EnergyBalanceInfo ComputeEnergyBalance(IEnumerable<OrganelleTemplate> organelles,
+        BiomeConditions biome, MembraneType membrane)
+    {
+        return ComputeEnergyBalance(organelles.Select(i => i.Definition), biome, membrane);
```

Comment:
Aside from the signature you changed already, shouldn't there be some consistency for the LINQ statements, which are sometimes `organelle => organelle.Definition` (see l.149 of this file) and sometimes `i => i.Definition`?

Reply:
I'm happy to change them around if you really want, but I think that anonymous functions have that liberty. I don't like "i" as an alias now that I look at it (this is inherited code I moved), but in the future I suggest we not try to tie every LINQ statement together like that. This isn't a data contract; it's intentionally limited to internal scope, specifically so we DONT have to worry about that.

Reply:
Latest styleguide and what I've done with ThriveDevCenter is to use the first letter of the variable name / class for the LINQ statement. So for example here all of these examples should be changed to be `o => o.Definition`

Reply:
Done, but I sure hope you don't expect me to enact this change across the codebase in this PR...

Reply:
I don't. I guess on my 100+ items TODO list it goes...